libbpf-tools/klockstat.c

```diff
@@ -3,6 +3,7 @@
  *
  * Based on klockstat from BCC by Jiri Olsa and others
  * 2021-10-26   Barret Rhoden   Created this.
+ * 07-Jan-2022  Rong Tao        Eliminate compilation warnings of reallocarray.
```

Comment:
We don't add a changelog entry for a change for eliminate compiler warnings. An entry can be added if the change is significant in functionality.

Reply:
> We don't add a changelog entry for a change for eliminate compiler warnings. An entry can be added if the change is significant in functionality.

Delete changelog in commit https://github.com/iovisor/bcc/pull/3798/commits/82dd76dd4fe866e7f911c47704d62c62f1288959